modules/optimizing-storage-azure.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: REFERENCE
+
+[id="optimizing-storage-azure_{context}"]
+= Optimizing storage performance for Microsoft Azure
+
+{product-title} and Kubernetes are sensitive to disk performance, and faster storage is recommended, particularly for etcd on the control plane nodes. 
+
+For production Azure clusters and clusters with intensive workloads, the VM OS disk should be able to sustain a tested and recommended minimum throughput of 5000 IOPS / 200MBps for control plane machines. 
+This throughput can be provided by having a minimum of 1 TiB Premium SSD (P30). 
+In Azure and Azure Stack Hub, disk performance is directly dependent on SSD disk sizes, so to achieve the throughput supported by `Standard_D8s_v3`, or other similar machine types available, and the target of 5000 IOPS, at least a P30 disk is required.
```

Comment:
This sentence has more than 32 words in it, and the name of an option shouldn't be used as a noun. Perhaps:

```suggestion
In Azure and Azure Stack Hub, disk performance is directly dependent on SSD disk sizes. To achieve the throughput supported by a `Standard_D8s_v3` virtual machine, or other similar machine types available, and the target of 5000 IOPS, at least a P30 disk is required.
```

Reply:
Resolved, thanks